modules/installation-creating-aws-control-plane.adoc

```diff
@@ -6,9 +6,14 @@
 [id="installation-creating-aws-control-plane_{context}"]
 = Creating the control plane machines in AWS
 
-You must create the control plane machines in Amazon Web Services (AWS) for your
-cluster to use. The easiest way to create these nodes is
-to modify the provided CloudFormation template.
+You must create the control plane machines in Amazon Web Services (AWS) for your cluster to use.
```

Comment:
Suggested edit for clarity:
s/for your cluster to use./that your cluster will use.

Reply:
Thank you, I have applied your suggestion.